dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/AbstractMethodProblem.java

```diff
@@ -17,12 +17,29 @@
 package com.google.cloud.tools.opensource.classpath;
 
 /**
- * The referenced {@code methodSymbol} is not implemented in the {@code targetClass} but the class
- * is declared to implement the method by {@code sourceClass}. Such unimplemented methods manifest
+ * The referenced {@code methodSymbol} is not implemented in the {@code sourceClass} but the class
+ * is declared to implement the method by {@code targetClass}. Such unimplemented methods manifest
```

Comment:
Switching source and target. Now implementation class is source because it references the interface.